src/Common/Description/GenericDescription.tsx

```diff
@@ -330,11 +241,14 @@ const GenericDescription = ({
             >
                 {isEditDescriptionView ? (
                     <div className="min-w-500 bcn-0 br-4 dc__border-top dc__border-left dc__border-right w-100 dc__border-bottom">
-                        <div className="pt-8 pb-8 pl-16 pr-16 dc__top-radius-4 flex bc-n50 dc__border-bottom h-36 fs-13">
-                            <div className="fw-6 lh-20 cn-9">Readme</div>
+                        <div className="pt-8 pb-8 pl-16 pr-16 dc__top-radius-4 flex bc-n50 dc__border-bottom h-36">
+                            <div className="flexbox dc__gap-6 dc__align-items-center">
+                                <UnorderedListIcon className="icon-dim-16" />
+                                <div className="fw-6 lh-20 cn-9 fs-13">{title ?? `Readme`}</div>
```

Comment:
Can write "Readme" and since we have constant file for this so move this text there.